### PR TITLE
AIRFLOW-1340 Create operator to copy data from S3 to Redshift

### PR DESCRIPTION
**Pull request by briancharous**
Dear Airflow maintainers,

Please accept this PR. I understand that it will not be reviewed until I have checked off all the steps below!

### JIRA
- [x] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/1340) issues and references them in the PR title.

### Description
- [x] Here are some details about my PR, including screenshots of any UI changes:
   * This PR adds a `S3ToRedshiftCopy` operator that is a wrapper around Redshift's `COPY` operation for loading data from files in S3.

### Tests
- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
  * `tests.operator.s3_to_redshift_operator`

### Commits
- [x] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### CODE REVIEW

**Comment by msumit** [comment]
@briancharous plz squash your commits. Also, make sure that the commit subject is limited to 50 chars.

**Comment by briancharous** [comment]
@msumit Squashed and reworded

File: airflow/operators/s3_to_redshift_operator.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
```
Comment (bolkedebruin) [inline airflow/operators/s3_to_redshift_operator.py:1]: Please move this to airflow/contrib/operators.

File: tests/operators/__init__.py [hunk 2]
```diff
@@ -20,4 +20,4 @@
 from .s3_to_hive_operator import *
 from .python_operator import *
 from .latest_only_operator import *
-
+from .s3_to_redshift_operator import *
```
Comment (bolkedebruin) [inline tests/operators/__init__.py:23]: to tests/contrib/operators plz

**Comment by Fokko** [comment]
@briancharous Could you close your PR, the functionality has been merged in: https://github.com/apache/incubator-airflow/pull/3161

**Pull request closed without merging**